.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - release-[7-9].*
```

Comment:
@DavidBoike does this mean that v10+ release branches will not build (my regex knowledge is awful)?

Reply:
It's not regex, it's a pretty simple wildcard syntax. But yes, it means that v10+ won't build. But long before we get to NServiceBus 10, Version 6 will fall out of extended support, and then we remove the TeamCity project, and this line becomes `release-*`.